lib/util/memcache.ex

```diff
@@ -6,10 +6,50 @@ defmodule Antikythera.Memcache do
   @moduledoc """
   In-memory cache for each executor pool.
 
-      iex> Antikythera.Memcache.write("foo", "bar", epool_id)
+  Antikythera provides in-memory cache controlled by AntikytheraCore for each executor pool.
```

Comment:
I think merging this statement with the above header line would be tidier. 
Also, "controlled by AntikytheraCore" is somewhat "in-depth", and not particularly required here IMO.

How about:

```ex
@moduledoc """
Easy-to-use in-memory cache for each executor pool.

Objects are stored in ETS internally. 
```

(Also, words on "per-instance"-ness could be stated around here, however it may be overkill since "stored in ETS" implies that. I want to hear from folks about this)

Reply:
Fixed in e8adcb50f657f23724421e7b0383d11d95fe9bb0

I think "in-memory cache" already implies that distributed nodes don't share cached records.